src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        System.out.println("\nSuccessfully added to list:\n" + addIndentation() + "[D]" + getStatusIcon() + " "
+                + description + " (by: " + by + ")");
```

Comment:
I don't think it's logical to place something related to adding to the list in the constructor for this class; would having it in its own `printAddToListMessage` method , or `Main`'s `execute` methods be a better place for it? (`Todo` and `Event` also have this output line.)

Reply:
Also, this repeats some of the string-ification logic you've already implemented in `toString`; I'd suggest calling that instead.

Reply:
Agree with @danbaterisna on the printline. If not for debugging purposes maybe can comment it?

Reply:
Thank you both for the advice, on hindsight adding this line to the constructor does not make sense. Will definitely move it to its own method.